mk/spksrc.service.mk

```diff
@@ -172,13 +172,11 @@ ifneq ($(strip $(SPK_USR_LOCAL_LINKS)),)
 	@jq --arg links_str '${SPK_USR_LOCAL_LINKS}' \
 		'."usr-local-linker" += ($$links_str | split (" ") | map(split(":")) | group_by(.[0]) | map({(.[0][0]) : map(.[1])}) | add )' $@ 1<>$@
 endif
-ifneq ($(strip $(SERVICE_WIZARD_SHARE)),)
-# e.g. SERVICE_WIZARD_SHARE=wizard_download_dir
-ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
```

Comment:
Can someone confirm that this doesn't break the existing DSM6 transmission?

Reply:
@publicarray on #4936 I have a solution that will not change the `SERVICE_WIZARD_SHARE` for DSM 6.

It uses a new Makefile variable `SERVICE_WIZARD_SHARE_RESOURCE` to use resource linker for shares with DSM < 7.

I will update kiwix to use the same variable after #4936 is merged.

Reply:
Thanks, I should have checked the other PR first.